coco/core.py

```diff
@@ -424,37 +420,15 @@ async def external_endpoint(self, request, endpoint):
         now = time.time()
         name = f"{os.getpid()}-{now}"
 
-        with await self.redis_async as r:
+        async with self.redis_async.client() as cli:
```

Comment:
Maybe call this variable something other than `cli`, just as that's more usually meaning a command line interface, which coco does have?